Engine/plugin/agsplugin.cpp

```diff
@@ -922,6 +931,19 @@ bool pl_use_builtin_plugin(EnginePlugin* apl)
 #endif // IOS_VERSION
 #endif // BUILTIN_PLUGINS
 
+    for(std::vector<InbuiltPluginDetails>::iterator it = _registered_builtin_plugins.begin(); it != _registered_builtin_plugins.end(); ++it) {
+        if (strncmp(apl->filename, it->filename, strlen(it->filename)) == 0) {
```

Comment:
Why is the comparison limited to length of the name in '_registered_builtin_plugins'? This means that if game requires plugin "MyPlugin", and someone registered plugin "My", that plugin will be used. Is this really a wanted behavior?